tempesta_fw/http.c

```diff
@@ -759,6 +759,9 @@ tfw_http_adjust_resp(TfwHttpResp *resp, TfwHttpReq *req)
 	if (r < 0)
 		return r;
 
+	/* Adjust @resp flags to use them in @tfw_connection_send */
+	hm->flags = (hm->flags & ~__TFW_HTTP_CONN_MASK) | conn_flg;
```

Comment:
Yeah, it seems the right place to set the flags from comment https://github.com/tempesta-tech/tempesta/pull/574/files#r69903646 .